### PR TITLE
Fixing misc test failures

### PR DESCRIPTION
**Pull request by galvana**
### Description Of Changes

Misc bug fixes for issues that made it to the `main` branch.

### Code Changes

* [ ] Added `defusedxml.ElementTree.*` to `pyproject.toml`'s `tool.mypy.overrides`
* [ ] Updated `verify_zip` function in `unsafe_file_utils` to resolve mypy errors
* [ ] Corrected the SVG file generated in the fixtures for `test_connector_template_loaders`, reading it as a string without base64 encoding it.

### Steps to Confirm

* [ ] _list any manual steps for reviewers to confirm the changes_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -39,8 +39,10 @@ module = [
   "botocore.*",
   "bson.*",
   "celery.*",
+  "citext.*",
```
Comment (galvana) [inline pyproject.toml:42]: Fixing order

File: pyproject.toml [hunk 2]
```diff
@@ -39,8 +39,10 @@ module = [
...
   "dask.*",
   "deepdiff.*",
+  "defusedxml.ElementTree.*",
```
Comment (galvana) [inline pyproject.toml:45]: This library doesn't have any type hints so I'm following the recommendation from mypy
https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker

File: tests/fixtures/saas_example_fixtures.py [hunk 3]
```diff
@@ -658,7 +658,7 @@ def planet_express_invalid_dataset() -> str:
 
 @pytest.fixture
 def planet_express_icon() -> str:
-    return encode_file_contents(
+    return load_as_string(
```
Comment (galvana) [inline tests/fixtures/saas_example_fixtures.py:661]: A recent changed exposed this mistake in the fixtures for the connector template tests. The SVG should be imported as a string without any encoding.

File: src/fides/api/util/unsafe_file_util.py [hunk 4]
```diff
@@ -51,7 +51,7 @@ def verify_zip(zip_file: ZipFile, max_file_size: Optional[int] = None) -> None:
         with zip_file.open(file_info) as file:
             # wraps the file read in an iterator that stops once no bytes
             # are returned or the max file size is reached
-            for chunk in iter(lambda: file.read(CHUNK_SIZE), b""):
+            for chunk in iter(lambda f=file: f.read(CHUNK_SIZE), b""):  # type: ignore
```
Comment (galvana) [inline src/fides/api/util/unsafe_file_util.py:54]: Bypasses mypy's `cell-var-from-loop` warning

**Review by adamsachs — APPROVED** [review]
looks good, thanks a bunch for jumping on these so quickly @galvana! 🙏

**Pull request merged**